README.md

```diff
@@ -26,7 +26,7 @@
 
 English | [简体中文](README-CN.md)
 
-[**Announcing Appwrite 1.2 with GraphQL support! Learn what's new!**](https://appwrite.io/graphql)
+<!-- [**Announcing Appwrite 1.3 with Database Relations! Learn what's new!**](https://appwrite.io/graphql) -->
```

Comment:
`Database Relations` or `Database Relationships`? Not sure whats official name.

Reply:
Let me know either way, I will double check articles

Reply:
```suggestion
<!-- [**Announcing Appwrite 1.3 with Database Relationships! Learn what's new!**](https://appwrite.io/graphql) -->
```